doniclient/osc/cli.py

```diff
@@ -368,9 +368,34 @@ def get_parser(self, prog_name):
             help="continue if an item already exists, rather than exiting",
             action="store_true",
         )
+        parser.add_argument(
+            "--force_update",
+            action="store_true",
+        )
         parser.add_argument("-f", "--file", help="JSON input file", type=FileType("r"))
         return parser
 
+    def get_patch(self, item):
```

Comment:
This has same code as `UpdateHardware` would it be better if this uses `UpdateHardware.get_patch()` or inherits `UpdateHardware`  so the code is not redundant.